src/display/annotation_layer.js

```diff
@@ -990,6 +993,7 @@ class WidgetAnnotationElement extends AnnotationElement {
     if (this.data.alternativeText) {
       this.container.title = this.data.alternativeText;
     }
+    this.containter.removeAttribute("tabInddsfsdfex");
```

Comment:
`this.containter === undefined", hence you're attempting to call `undefined.removeAttribute(...)` which should throw :-)

Besides, this line no longer ought to be necessary with the change above

Reply:
Arff... I just forgot to remove the line....